app/src/main/java/com/google/android/ground/ui/datacollection/tasks/polygon/DrawAreaTaskFragment.kt

```diff
@@ -112,7 +112,7 @@ class DrawAreaTaskFragment @Inject constructor() : AbstractTaskFragment<DrawArea
     viewModel.polygonArea.observe(
       viewLifecycleOwner,
       { area ->
-        val formattedArea = String.format(Locale.US, "%.2f", area)
+        val formattedArea = String.format(Locale.getDefault(), "%.2f", area)
```

Comment:
Can you just put the `%.2f` inside `R.string.area_message` instead of formatting it explicitly here?